fix: make the Content-Type check spec compliant (#844)
tauri-apps/wry: .changes/fix-android-content-type-check.md

---
"wry": patch
---

> Make the `Content-Type` spec compliant. This fixes the injection of `intialization_scripts` for devServers where the `Content-Type` header includes more information than just `"text/plain"`.

Reviewer: ```suggestion
On Android, fix the injection of `intialization_scripts` for devServers where the `Content-Type` header includes more information than just `"text/plain"`.
```
